manage_breast_screening/core/jinja2/components/appointment-header/template.jinja

```diff
@@ -10,3 +12,16 @@
 {% endif %}
 
 <p class="nhsuk-u-margin-bottom-4"><a href="{{ appointment.participant_url }}">View participant record</a></p>
+
+{% if appointment.can_be_started_by(user) %}
```

Comment:
Should we use `presented_appointment` here (and in the rest of this template) for consistency with other template code?